app/code/Magento/Sales/i18n/en_US.csv

```diff
@@ -1,801 +1,801 @@
-"Credit Memos","Credit Memos"
```

Comment:
Could you discard these changes and leave only required translate.

Reply:
@swnsma - I didn't get this. do you want me to remove this entry from csv?

Reply:
@rahulwebkul,
In this file are 1,288 changed lines (644 lines have been changed).
I expected to see only 1 line added.

Reply:
@swnsma - csv file updated. please review the changes